tests/utils/test_cases.py

```diff
@@ -242,32 +242,3 @@ def test_case_8() -> Collection:
                 "data-files/catalogs/" "planet-example-v1.0.0-beta.2/collection.json"
             )
         )
-
-    @staticmethod
-    def test_case_9() -> Catalog:
```

Comment:
Inlined the creation of this catalog as it's specific to that issue, the "test_case_X" methods are meant to represent a set of reusable test Catalogs that are used through many tests.